cookbook/how_to_upgrade_to_new_version.rst

```diff
@@ -0,0 +1,79 @@
+.. index::
+    single: Upgrade
+
+How to upgrade to new version
+==============================
+
+1. Checkout from the GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve new version and upgrade your ORO CRM instance, please execute the next steps:
+
+**1** Create backups of your Database and Code.
+
+**2** "Cd" to crm root folder and Pull changes from repository
+
+.. code-block:: bash
+
+$ cd cd /path/to/crm_folder
+$ git pull
+$ git checkout <VERSION TO UPGRADE>
+
+**3** Upgrade composer dependency
+
+.. code-block:: bash
+
+$ php composer.phar install --prefer-dist --no-dev
+
+**4** Remove old caches and assets
+
+.. code-block:: bash
+
+$ rm -rf app/cache/*
+$ rm -rf web/js/*
+$ rm -rf web/css/*
+
+**5** Upgrade platform
+
+.. code-block:: bash
+
+php app/console oro:platform:update --env=prod --force
+
+2. Download the Source Code Archive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve new version and upgrade your ORO CRM instance, please execute the next steps:
```

Comment:
Edit: To retrieve a new version and upgrade your ORO CRM instance, please execute the following steps: